### PR TITLE
Subscribe API content updates - Closes #549

### LINKED ISSUE #549: Subscribe API content updates
### Actual behavior

The events `update.forgers` and `update.transactions.confirmed` are not returning properties as described in the documentation.

### Expected behavior

Make them work properly as described in the documentation.

https://github.com/LiskHQ/lisk-service/blob/development/docs/api/websocket_subscribe_api.md

### Steps to reproduce

Run the subscriber against `wss://betanet5-service.lisk.io/blockchain`

### Which version(s) does this affect? (Environment, OS, etc...)

`0.3.0` on `betanet`

### Additional information

```
====== update.forgers ======
{}

====== update.transactions.confirmed ======
{
  "data": [
    {
      "isFulfilled": false,
      "isRejected": true,
      "rejectionReason": {}
    }
  ],
  "meta": {
    "offset": 0,
    "count": 1,
    "total": 1
  }
}
```

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #549 

### How was it solved?
- [x] Fix `update.forgers`
- [x] Fix `update.transactions.confirmed`
- [x] Events works properly as described in the documentation.

### How was it tested?
- [x] Local

### CODE REVIEW

**Review by sameersubudhi — APPROVED** [review]
LGTM and works as expected! 👍

File: services/gateway/app.js [hunk 1]
```diff
@@ -129,13 +131,16 @@ broker.createService({
...
+			meta: metaResponse,
+		})),
+		'transactions.confirmed': (payload) => sendSocketIoEvent('update.transactions.confirmed', mapper(payload, {
+			data: ['data', transactionResponse],
+			meta: metaResponse,
 		})),
-		'transactions.confirmed': (payload) => sendSocketIoEvent('update.transactions.confirmed', payload),
+		'round.change': (payload) => sendSocketIoEvent('update.round', payload),
+		'forgers.change': (payload) => sendSocketIoEvent('update.forgers', payload),
 		'update.fee_estimates': (payload) => sendSocketIoEvent('update.fee_estimates', payload),
```
Comment (MichalTuleja) [inline services/gateway/app.js:144]: Taking the chance we're here, could you also take care of those mappings please?

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: services/gateway/app.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/gateway/app.js:144]: Sure, I will do that.

**Review by MichalTuleja — APPROVED** [review]
Looks great, thanks @nagdahimanshu 👍

**Pull request merged**